src/hooks/api/useElectronicAccess.js

```diff
@@ -0,0 +1,26 @@
+import { useOkapiKy } from '@folio/stripes/core';
+import { useQuery } from 'react-query';
+
+export const useElectronicAccessRelationships = (options = {}) => {
+  const ky = useOkapiKy();
+
+  const { data, isLoading: isElectronicAccessLoading } = useQuery(
+    {
+      queryKey: 'electronicAccess',
+      cacheTime: Infinity,
+      staleTime: Infinity,
+      queryFn: () => ky.get('electronic-access-relationships').json(),
+      ...options,
+    },
+  );
+
+  const electronicAccessRelationships = data?.electronicAccessRelationships?.map((item) => ({
+    label: item.name,
+    value: item.id,
+  })) || [];
```

Comment:
wrap this into `useMemo` to avoid triggering unnecessary recalculations on each re-render.

Reply:
@mariia-aloshyna  it's not required, because in this case memoizing can cost more then potential perfomance improvements. 
`cacheTime: Infinity,
      staleTime: Infinity,` 
     means that this will be executed only once + count of potential reletionships are very low.